packages/commcare/test/index.js

```diff
@@ -97,3 +96,74 @@ describe('SubmitXls', () => {
     expect(formdata.get('create_new_cases')).to.equal('on');
   });
 });
+
+describe('getCases', () => {
+  it('should fetch cases', async () => {
+    testServer
+      .intercept({
+        path: `/a/${domain}/api/v0.5/case`,
+        method: 'GET',
+      })
+      .reply(200, () => {
+        // simulate a return from commcare
+        return {
+          data: [
```

Comment:
I would add a `meta` key as well and check that the returned meta gets written to the right part of state. 

The value of these tests is to ensure that, given a URL from the user and some response data from commcare, the `get` function produces a state object with the right shape.

And actually we probably should have an assertion on `state.response` - just check the statusCode or something

Reply:
Oh hang on this still isn't right - we need to return `{ meta, objects }` here